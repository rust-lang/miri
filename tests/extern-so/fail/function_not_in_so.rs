//@only-target-linux
//@only-on-host

extern "C" {
    fn foo();
}

fn main() {
    unsafe {
        foo(); //~ ERROR: unsupported operation: can't call foreign function: foo
    }
}
