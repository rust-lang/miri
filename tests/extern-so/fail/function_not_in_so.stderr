error: unsupported operation: can't call foreign function: foo
  --> $DIR/function_not_in_so.rs:LL:CC
   |
LL |         foo();
   |         ^^^^^ can't call foreign function: foo
   |
   = help: this is likely not a bug in the program; it indicates that the program performed an operation that the interpreter does not support
   = note: BACKTRACE:
   = note: inside `main` at $DIR/function_not_in_so.rs:LL:CC

note: some details are omitted, run with `MIRIFLAGS=-Zmiri-backtrace=full` for a verbose backtrace

error: aborting due to previous error

